docs/tasks/includes/_rust-read.md

```diff
@@ -0,0 +1,28 @@
+
+Use the [`Reader`](https://docs.rs/c2pa/latest/c2pa/struct.Reader.html) struct to read manifest data from a file or stream.
+
+### Reading from a file
+
+Use [`from_file`](https://docs.rs/c2pa/latest/c2pa/struct.Reader.html#method.from_file) to read manifest data from a file:
+
+```rust
+use c2pa::Reader;
```

Comment:
Spacing between imports and code?

Reply:
Fixed, if I understand your comment correctly.